package.json

```diff
@@ -14,13 +14,13 @@
   "engines": {
     "node": ">=6.14.2"
   },
-  "libui": "alpha3.5-master-003",
+  "libui": "alpha4.1",
   "scripts": {
     "docs": "node tools/docs.js",
     "start": "node --expose-gc example.js",
     "download-libui": "node tools/libui-download.js",
     "download-libui-napi": "node tools/libui-napi-download.js",
-    "clean": "rimraf build *.h *.node libui.* node_modules",
+    "clean": "rimraf build *.h *.node libui.* node_modules/!(rimraf|.bin)",
```

Comment:
Why does this delete the node_modules folder?

Reply:
I had an issue on Windows: executing rimraf locks it's binary file, so WIndows is not able to delete it. If we don't exclude it, the scripts fails with an error.

Reply:
My question was: Why delete node_modules at all?
Is `clean` meant as the counterpart to `npm run install` or `npm install`?

Reply:
> What's the difference between the two?

`npm install`: Install dependencies and run the `install` script from package.json
`npm run install`: Only run the `install` script from package.json

Reply:
Oh I didn't know that! Nice!